jena-arq/src/main/java/org/apache/jena/riot/lang/JsonLDReader.java

```diff
@@ -45,9 +45,20 @@
 import org.apache.jena.riot.system.* ;
 import org.apache.jena.sparql.core.Quad ;
 import org.apache.jena.sparql.util.Context ;
+import org.apache.jena.sparql.util.Symbol;
 
 public class JsonLDReader implements ReaderRIOT
 {
+    private static final String SYMBOLS_NS = "http://jena.apache.org/riot/jsonld#" ;
+    /**
+     * Symbol to use to pass the "@context" to be used when reading the jsonld
+     * (overriding the actual @context in the jsonld)
+     * Expected value: the value of the "@context", 
+     * as expected by the JSONLD-java API (a Map) */
+    public static final Symbol JSONLD_CONTEXT = Symbol.create(SYMBOLS_NS + "JSONLD_CONTEXT");
```

Comment:
Ideally, the constants go in `RIOT` so that application code just needs to import RIOT.  There aren't many - only an odd legacy feature - because Turtle/NTriples like writers don't have lots of configuration control.